src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -162,6 +162,12 @@ internal sealed class MeterProviderSdk : MeterProvider
                             try
                             {
                                 metricStreamConfig = viewConfig(instrument);
+
+                                if (metricStreamConfig is ExplicitBucketHistogramConfiguration
+                                    && instrument.GetType().GetGenericTypeDefinition() != typeof(Histogram<>))
+                                {
+                                    metricStreamConfig = null;
```

Comment:
Q: Should we log an event for this to let the user know they might have made a mistake?

Reply:
`OpenTelemetrySdkEventSource.Log.MetricViewIgnored` See! I added this predicting future demand 😆